apps/meteor/tests/e2e/page-objects/fragments/home-content.ts

```diff
@@ -156,6 +156,14 @@ export class HomeContent {
 		return this.page.locator('#modal-root .rcx-button-group--align-end .rcx-button--primary');
 	}
 
+	get btnModalConfirmDelete(): Locator {
+		return this.page.getByRole('button', { name: 'Yes, delete', exact: true });
+	}
+
+	get btnCancelQuotePreview(): Locator {
+		return this.page.getByRole('button').locator('.rcx-icon--name-cross');
```

Comment:
Is this the best way we have to find this button? If so it is a sign the the ui is not accessible enough 🤔